src/v2/Apps/Collect/collectRoutes.tsx

```diff
@@ -98,6 +102,23 @@ export const collectRoutes: RouteConfig[] = [
     },
     prepareVariables: initializeVariablesWithFilterState,
     query: CollectionAppQuery,
+    render: ({ Component, props, match }) => {
+      if (Component && props) {
+        const { context, params } = match
+        const { user } = context
+        const { slug } = params
+
+        if (userHasLabFeature(user, "Artist Series")) {
```

Comment:
I forget exactly at which point in this router code we attempt to fetch for a collection, but will we still hit this code-path in the case where a collection doesn't exist? That's the main use-case for this-- we need to route to the new artist series once we've deleted the collection in KAWS. (Might be fine, just want to check!)

Reply:
Yea, this would still get hit when the MP query resolves. Since we plan on fully deleting the collections from KAWS, I can move this out of the found stack entirely, and into a more standard Express middleware.